lmp-bsp.xml

```diff
@@ -5,12 +5,12 @@
 
   <default remote="lmp-mirrors" revision="master" sync-j="4"/>
 
-  <project name="meta-arm" path="layers/meta-arm" revision="260e3adc2bf322f52d81c0642c825088a88bb051"/>
-  <project name="meta-freescale" path="layers/meta-freescale" revision="84be484645cdcc12bc12591874bfebd24cc0c9a6"/>
-  <project name="meta-freescale-3rdparty" path="layers/meta-freescale-3rdparty" revision="8b61684f0b1ba8bacdf3a69d993445e9791d4932"/>
-  <project name="meta-intel" path="layers/meta-intel" revision="5cfefd9a8ff1f5a3534c1ba9d7d7f6971ed5d56f"/>
-  <project name="meta-raspberrypi" path="layers/meta-raspberrypi" revision="e6d2ff0b0cbb925157c95b07327c2a7dc145fabe"/>
-  <project name="meta-yocto" path="layers/meta-yocto" revision="c4c74d1e575217ddc4b74759cd83186a70940ef9"/>
-  <project name="meta-tegra" path="layers/meta-tegra" revision="d47fa3649b6155b9c01e06087917ddca341e7cf1"/>
-  <project name="meta-ti" path="layers/meta-ti" revision="1de40ea7ee8ff136dce163b4ad1c1eddf35852a7"/>
+  <project name="meta-arm" path="layers/meta-arm" revision="a8fe9d22eaefc294f91096c6a32663e2f4ab3b10"/>
+  <project name="meta-freescale" path="layers/meta-freescale" revision="d015b23d0ac89c744557848a8bce79729dc9e351"/>
+  <project name="meta-freescale-3rdparty" path="layers/meta-freescale-3rdparty" revision="1dfc65dd2006b51d156be5bcda0e3c72c936ad0a"/>
+  <project name="meta-intel" path="layers/meta-intel" revision="0be954e82f09d235d2b5e5ea01bb62ffa8d0b5f7"/>
+  <project name="meta-raspberrypi" path="layers/meta-raspberrypi" revision="6df7e028a2b7b2d8cab0745dc0ed2eebc3742a17"/>
+  <project name="meta-yocto" path="layers/meta-yocto" revision="bd166d1fb8dc1bed7e71bd06b970a3da9149203e"/>
+  <project name="meta-tegra" path="layers/meta-tegra" revision="3683a5aae81f73d26e354eae16fea1830cbb642e"/>
```

Comment:
I think we should probably use `lmp-mirrors/scarthgap-l4t-r35.x` for meta-tegra

Reply:
For now we are using the scarthgap-l4t-r35.x https://github.com/OE4T/meta-tegra/commit/3683a5aae81f73d26e354eae16fea1830cbb642e that supports Xavier SoC, the latest jetpack is on the meta-tegra/scarthgap and the Xavier is not supported there.
We can update later to latest jetpack and move the meta-tegra/scarthgap-l4t-r35.x to some custom manifest using a partner factory or something like that.